### PR TITLE
Correct project name in LICENSE and update year

### PR DESCRIPTION
**Pull request by KazW**
Figured I'd update the year as well since it's such a small commit.

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2018, Richard Feldman
+Copyright (c) 2019, Richard Feldman
```
Comment (rtfeldman) [inline LICENSE]: Something I learned recently is that [it's best practice to leave the oldest applicable copyright date](https://stackoverflow.com/a/2391555); updating it actually weakens the copyright claim!

```suggestion
Copyright (c) 2018, Richard Feldman
```

File: LICENSE [hunk 2]
```diff
@@ -11,7 +11,7 @@ modification, are permitted provided that the following conditions are met:
   this list of conditions and the following disclaimer in the documentation
   and/or other materials provided with the distribution.
 
-* Neither the name of elm-css nor the names of its
+* Neither the name of elm-iso8601-date-strings nor the names of its
```
Comment (rtfeldman) [inline LICENSE]: Good catch! 😅

File: LICENSE (hunk 1 above)
Comment (KazW) [reply LICENSE]: Interesting, makes sense though.

**Comment by rtfeldman** [comment]
Lovely, thanks @KazW!

**Pull request merged**